src/main/java/org/candlepin/subscriptions/metering/service/prometheus/task/PrometheusMetricsTaskManager.java

```diff
@@ -89,24 +91,41 @@ private void queueMetricUpdateForAccount(
   @Transactional
   public void updateMetricsForAllAccounts(
       String productTag, OffsetDateTime start, OffsetDateTime end) {
+    updateMetricsForAllAccounts(productTag, start, end, Optional.empty());
```

Comment:
Instead of using optional here, you could just use `RetryTemplate.builder().maxAttempts(1).build()`, then that would avoid the need to have the if statement. Thoughts?

Reply:
:+1:

Reply:
DONE